### PR TITLE
(freecad) adding erroraction continue

### PR DESCRIPTION
**Pull request by RedBaron2**
This will allow this package to pass verification
@AdmiringWorm @majkinetor @gep13 @pascalberger 

This package [.portable](https://chocolatey.org/packages/freecad.portable) has been stuck in verification limbo since April 19th, 2020. I had asked if this or that should be done to get this package to pass verification. The answer was always not familiar with it will get back to you.

## Description

The use of the Get-Version cmdlet causes grief if the package AU is not installed. Since all Core Packages have `$ErrorActionPreference = 'Stop';` This was causing a **_HARD Stop_** and thus causing the verification to Fail.
The cmdlet of `Get-Command` has `-erroraction` as a choice of parameters. Setting this to Continue will allow the package to complete the if..else lines

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [x] Issue 1 [link](https://gist.github.com/0aadf010f605a2f7214b39c1d6da8bde)
  - [ ] Issue 2 Link
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/freecad/tools/helper.ps1 [hunk 1]
```diff
@@ -7,7 +7,7 @@ param(
 )
 $fileName = ((($thePackage.url64 -split('/'))[-1]) -replace( "\.$($thePackage.fileType)", '' ) )
  if ($fileName -match "portable") { 
-	if (Get-Command Get-Version){ $version = ( Get-Version $fileName ).Version;
+	if (Get-Command Get-Version -ErrorAction "Continue"){ $version = ( Get-Version $fileName ).Version;
```
Comment (majkinetor) [inline automatic/freecad/tools/helper.ps1:10]: `Get-Version` is AU function that has no relevance to this package. You want  something like `Get-Item  $filePath | % VersionInfo | % FileVersion`

Comment (RedBaron2) [reply automatic/freecad/tools/helper.ps1:10]: @majkinetor 
Yes, this is a AU function. This is a Check to see if AU is installed on the system, and if so use the `Get-Version` function.  The package of FreeCAD _does not_ need **AU** as a **_dependency_**. If you want me to eliminate the `Get-Version` check I will do that option.

Comment (majkinetor) [reply automatic/freecad/tools/helper.ps1:10]: Yes I know but its completelly wrong. Get-Version doesn't do that. Not sure it got there.

**Comment by RedBaron2** [comment]
@majkinetor 
I have removed the Get-Version check all together. It really wasn't needed all all. Thanks for the review

**Pull request merged**